monkestation/code/modules/replays/subsystem/replay.dm

```diff
@@ -23,125 +23,81 @@ SUBSYSTEM_DEF(demo)
 	var/last_written_time = null
 	var/last_chat_message = null
 
-	// stats stuff
-	var/last_queued = 0
-	var/last_completed = 0
-
-/datum/controller/subsystem/demo/proc/write_time()
-	var/new_time = world.time
-	if(last_written_time != new_time)
-		if(initialized)
-			WRITE_LOG_NO_FORMAT(GLOB.demo_log, "time [new_time]\n")
-		else
-			pre_init_lines += "time [new_time]"
-	last_written_time = new_time
-
-/datum/controller/subsystem/demo/proc/write_event_line(line)
-	write_time()
-	if(initialized)
-		WRITE_LOG_NO_FORMAT(GLOB.demo_log, "[line]\n")
-	else
-		pre_init_lines += line
+	var/disabled = FALSE
```

Comment:
I'm adding this var instead of just using `can_fire` like I did previously so it can be manually paused without _disabling_ it